packages/tokens/build.js

```diff
@@ -60,6 +120,36 @@ StyleDictionary.registerFormat({
   },
 })
 
+/** Creates named type declaration for Themes. Allows for TypeScript autocomplete */
+StyleDictionary.registerFormat({
+  name: 'typescript/es6-declarations/theme',
+  formatter: function (dictionary) {
+    let declaration = 'export declare type Theme = {\n'
+    dictionary.allProperties.forEach((token) => {
+      declaration += `  ${stripMode(token.name)}: string;\n`
+    })
+
+    declaration += `}
+
+export declare const themes: {
```

Comment:
Also nitpicky, but this one is especially goofy looking. May be worth doing separately:
```
})
declaration += `}\n\n`

declaration += `export...
```